### PR TITLE
 Drop adsimport before create it

### PR DESCRIPTION
**Pull request by oliverds**

### CODE REVIEW

File: oc/classes/controller/panel/import.php [hunk 1]
```diff
@@ -461,10 +461,23 @@ private function insert_into_import($ads_array,$expected_header)
             }
         }
 
+        //drop table first
+        $query = DB::query(Database::INSERT, 'DROP TABLE IF EXISTS `' . $prefix . 'adsimport`;');
```
Comment (neo22s) [inline oc/classes/controller/panel/import.php]: but doing this... if you want to import many ads... will delete those ads you already imported... damn. maybe check if table exists and has same fields? ufff

Comment (oliverds) [reply oc/classes/controller/panel/import.php]: But you process queue first no?... anyway check if the table exists and doesn't have same columns before drop it.

Comment (neo22s) [reply oc/classes/controller/panel/import.php]: no, thers a queue and then you click the process button. no? or maybe has changed? hahaha

Comment (oliverds) [reply oc/classes/controller/panel/import.php]: Hahah. Nop nothing has changed... Upload csv file -> process queue button -> ads imported :)

When csv file is uploaded we create the adsimport table

Comment (neo22s) [reply oc/classes/controller/panel/import.php]: image

you need to click process,  no? :O

**Pull request merged**